### DISCUSSION TITLE
Custom domain

### DISCUSSION THREAD

**Discussion by success353** [vercel/community#1408] [Help]
### Summary

Hi vercel, we are working on a project with Nextjs and are using vercel for deployment. We want to add a feature that allows users to connect a custom domain to their site. Please help, thanks

**Comment by swarnava** [comment]
Hi,

Sounds like you are looking something like this https://vercel.com/blog/wildcard-domains#demo-time

**Comment by success353** [comment]
Thanks for the response, but this does not answer my question. I want them to be able to connect a domain not to create a subdomain
